src/utils/utils.ts

```diff
@@ -29,15 +29,56 @@ export const imageURL = (
   return `https://iiif.nypl.org/iiif/2/${imageId}/${region}/${size}/${rotation}/default.jpg`;
 };
 
+/**
+ * Returns the number of digitized items in repo api.
+ */
+
+export const getNumDigitizedItems = async () => {
+  const apiUrl = `${process.env.API_URL}/api/v2/items/total`;
+  const res = await apiCall(apiUrl);
+  return addCommas(res.count.$) || 0;
```

Comment:
If there is an error with the Repo API, the fallback will render as "explore 0 items digitized ...". What if the fallback is `863848`? (But set before it's passed to `addCommas`, e.g. `const fallbackCount = 863848; const totalItems = res.count.$ || fallbackCount;`)